docs/tasks/issuers/setup-acme/dns01/index.rst

```diff
@@ -62,6 +62,13 @@ Example usage::
 
     --dns01-recursive-nameservers "8.8.8.8:53,1.1.1.1:53"
 
+If you're using the `cert-manager` helm chart, you can set recursive nameservers
+through `.Values.extraArgs` or at the command at helm install/upgrade time
+with `--set`:
+
+    --set 'extraArgs={--dns01-recursive-nameservers=8.8.8.8:53\,1.1.1.1:53}'
```

Comment:
It seems the backslash gets eaten by GitHub's HTML view of this .rst file. As I explained [here](https://github.com/jetstack/cert-manager/issues/1163#issuecomment-484171354), the backslash is important for this to work.